CHANGELOG.md

```diff
@@ -4,6 +4,9 @@
 ### Added
 - Command Generate - Generate MongoDB HasMany.
 
+### Changed
+- Technical - Updated error message when wrong password
```

Comment:
```suggestion
- Authentication - Updated warning message when authentication failed.
```

Some issues with the actual text:
- I think it's not a problem about password (email and/or token I guess).
- It's a warning (not an error).
- We (currently) have to add a trailing dot.
- We try to avoid "technical" when it has an impact on users.
- TODO: Don't forget to update the PR title too (and the commit message when squash+merging).

Maybe my suggestion is not that good, feel free to suggest a better one!

Reply:
The warning appears when the user enters a "wrong/ not well formatted email", since this PR is fixing this issue, I think `Authentication - Updated warning message when the email is not well formatted` is better :)